File: javascript/Day2/Task4.js
function fuelChecker(gallon){
	let distanceCanBeCovered = gallon * 25;
	let fillingStationDistance = 50;
	
	if (distanceCanBeCovered == fillingStationDistance){
		return "True"}
	else if (distanceCanBeCovered != fillingStationDistance){
		return "False"}
}

console.log(fuelChecker(2));

File: python/Day2/Task4.py
def fuelChecker(gallon):
        distanceCanBeCovered = gallon * 25;
        fillingStationDistance = 50;

        if (distanceCanBeCovered == fillingStationDistance):
        	return "True"
        elif (distanceCanBeCovered != fillingStationDistance):
                return "False"

print(fuelChecker(2))
